src/main/resources/schema.sql

```diff
@@ -1,4 +1,4 @@
---TODO: DROP TABLE IF EXISTS program_county;
+DROP TABLE IF EXISTS program_county;
```

Comment:
As shown in the TODO comment, you need to drop the join table first or else there will be records referencing the program/county tables